pkg/sources/github/github.go

```diff
@@ -150,6 +155,76 @@ func (s *Source) Init(aCtx context.Context, name string, jobID, sourceID int64,
 	return nil
 }
 
+func (s *Source) isPublic(repoURL string) (visability source_metadatapb.Visibility) {
```

Comment:
**Nit (optional):** A better name might be `visibilityOf` or `getVisibility` since this method is no longer returning a boolean.